src/nvim/keymap.h

```diff
@@ -220,7 +220,7 @@ enum key_extra {
   KE_KINS = 79,             // keypad Insert key
   KE_KDEL = 80,              // keypad Delete key
 
-  KE_CSI = 81,              // CSI typed directly
+  // KE_CSI = 81,
```

Comment:
perhaps some comment text to this like "disabled as this key cannot be typed in neovim UI:s".

Reply:
I'll add "Nvim doesn't need escaping CSI".